### ISSUE TITLE
npm package ?

### ISSUE THREAD

**Issue by jerefrer** [BreadMaker/semantic-ui-daterangepicker#5]
First, thanks for this awesome daterangepicker, it's looks very neat !

I would have loved to use it in my projet but unfortunately it proved too complicated for me to include it in this new stack I'm trying and not yet comfortable with (webpack, npm, react, es6 with babel, ...).

It would have been way simpler if it was available as an npm package, or even a bower package.

And I think this would be a good idea no matter what :)

Cheers !

**Comment by tgolen** [comment]
I forked this repo and published it to NPM.

https://github.com/tgolen/semantic-ui-daterangepicker

**Comment by john-osullivan** [comment]
Thanks for publishing to npm, @tgolen .  @BreadMaker , do you have any plans to submit this repo yourself?  This repo is already a fork from the main dev effort on the bootstrap datepicker, splitting into *another* new repo just to publish to NPM is almost certain to prevent consistent code.  Thoughts?

**Comment by BreadMaker** [comment]
Yes, @john-osullivan, this MUST be a proper separate package published from here. As you may be aware, I'm refactoring everything again, in order to acomplish that. You can see that work in the [`fresh-start`](https://github.com/BreadMaker/semantic-ui-daterangepicker/tree/fresh-start) branch.
